x-pack/plugins/apm/public/components/app/RumDashboard/CoreVitals/index.tsx

```diff
@@ -20,10 +20,11 @@ const CoreVitalsThresholds = {
 export function CoreVitals() {
   const { urlParams, uiFilters } = useUrlParams();
```

Comment:
Should we rename this to make the distinction between path params (that we get from react-router) and query params clear?

```suggestion
  const { queryParams, uiFilters } = useQueryParams();
```

Reply:
I'd be fine with doing that but we would have to change it in a hell of a lot of places. I'd prefer doing that change in the next round of changes and leaving it for now.